gtsfm/retriever/retriever_base.py

```diff
@@ -0,0 +1,53 @@
+"""Base class for the Retriever, which provides a list of potential image pairs.
+
+Authors: John Lambert
+"""
+
+import abc
+from enum import Enum
+from typing import List, Tuple
+
+import dask
+from dask.delayed import Delayed
+
+from gtsfm.loader.loader_base import LoaderBase
+
+
+class ImageMatchingRegime(str, Enum):
+    SEQUENTIAL: str = "sequential"
+    RETRIEVAL: str = "retrieval"
+    EXHAUSTIVE: str = "exhaustive"
+    SEQUENTIAL_WITH_RETRIEVAL: str = "sequential_with_retrieval"
```

Comment:
could you use all caps, matching with the enum? that is how it has been done in a few other places, just for uniformity.

Reply:
yeah, the argparse argument is of type string here: https://github.com/borglab/gtsfm/pull/453/files#diff-dabbe43e473aeebd439bd77b3d2cafbd366a4da78dbc171661d01e9e9a2f2142R70

Reply:
You could have the flag as lower case and convert it to upper before calling the enum (that way the user can use whatever case), and the enum is still consistent.